data/brands/office/insurance.json

```diff
@@ -42,6 +42,19 @@
         "short_name": "AAA"
       }
     },
+    {
+      "displayName": "Aésio Mutuelle",
+      "locationSet": {"include": ["fr"]},
+      "tags": {
+        "brand": "Aésio mutuelle",
+        "brand:wikidata": "Q56320205",
+        "brand:wikipedia": "fr:Aésio Mutuelle",
+        "name": "Aésio mutuelle",
+        "office": "insurance",
+        "short_name": "Aésio",
+        "website": "https://www.aesio.fr/"
```

Comment:
1. `website` means the website for a single office, but `brand:website` means the website for all this brand share.
2. the brand's website information can be access from wikidata, so don't need to add it in NSI.

Reply:
```suggestion
```